src/main/java/frc/trigon/robot/subsystems/roller/RollerConstants.java

```diff
@@ -0,0 +1,23 @@
+package frc.trigon.robot.subsystems.roller;
+
+import frc.trigon.robot.utilities.SpeedMechanism2d;
+
+public class RollerConstants {
+    public static final double GEAR_RATIO = 1;
+    private static final double MAX_DISPLAYABLE_VELOCITY = 4;
+    static final SpeedMechanism2d ROLLER_MECHANISM = new SpeedMechanism2d("Mechanisms/RollerMechanism", MAX_DISPLAYABLE_VELOCITY);
+
+    public enum RollerState {
+        STOPPED(0),
+        COLLECTION(0),
+        FEEDING(0),
+        SCORE_AMP(0),
+        SCORE_TRAP(0);
```

Comment:
give these example numbers. 
Give the collection something like -30
Feeding -40
Scoring for the amp, 30 
Scoring for the trap, 30 
(All of these can and most likely will change)